src/api/services/auth_service.py

```
@@ -0,0 +1,167 @@
+from datetime import datetime, timedelta
+import pyotp
+import bcrypt
+import uuid
+import jwt
+from flask import current_app
+from flask_jwt_extended import create_access_token, create_refresh_token
```

Ruff (0.11.9)

`flask_jwt_extended.create_access_token` imported but unused

Remove unused import

(F401)